modules/preparing-an-inital-cluster-deployment-for-mce.adoc

```diff
@@ -0,0 +1,401 @@
+The AI service requires persistent volumes (PVs), which is installed through the {product-title} Local Storage Operator.
+====
+
+. For AI service, create the following manifests and save in the `<manifests_directory>/openshift` folder:
+
++
+.Example `lso_namespace.yaml`
+
++
+[source,yaml]
+----
+  apiVersion: v1
+  kind: Namespace
+  metadata:
+    annotations:
+      openshift.io/cluster-monitoring: "true"
+    name: openshift-local-storage
+----
+
++
+.Example `lso_operatorgroup.yaml`
+
++
+[source,yaml]
+----
+  apiVersion: operators.coreos.com/v1
+  kind: OperatorGroup
+  metadata:
+    name: local-operator-group
+    namespace: openshift-local-storage
+  spec:
+    targetNamespaces:
+      - openshift-local-storage
+----
+
++
+.Example `lso_subscription.yaml`
+
++
+[source,yaml]
+----
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: local-storage-operator
+    namespace: openshift-local-storage
+  spec:
+    installPlanApproval: Automatic
+    name: local-storage-operator
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
+----
++
+[NOTE]
+====
+After creating all the manifests, your Filesystem must display as follows:
+
+.Example Filesystem
+
+[source,terminal]
+----
+<manifests_directory>
+    ├─ install-config.yaml
+    ├─ agent-config.yaml
+    └─ /openshift
+        ├─ mce_namespace.yaml
+        ├─ mce_operatorgroup.yaml
+        ├─ mce_subscription.yaml
+        ├─ lso_namespace.yaml
+        ├─ lso_operatorgroup.yaml
+        └─ lso_subscription.yaml
+----
+====
+
+. Create the `agent.iso` image by running the following command:
+
++
+[source,terminal]
+----
+$ openshift-install agent create image --dir <manifests_directory>
+----
+
+. Once the image is ready , boot the target machine and wait for the installation to complete.
+
+. To monitor the installation, run the following command:
++
+[source,terminal]
+----
+$ openshift-install agent wait-for install-complete --dir <manifests_directory>
+----
+
++
+[NOTE]
+====
+To configure a fully functional hub cluster,you must manually create the following manifests.
+The order of the manifest creation is important and where required, the waiting condition is displayed.
+====
+
+. For the PVs that are required by the AI service, create the following manifests:
+
++
+[source,yaml]
+----
+  apiVersion: local.storage.openshift.io/v1
+  kind: LocalVolume
+  metadata:
+   name: assisted-service
+   namespace: openshift-local-storage
+  spec:
+   logLevel: Normal
+   managementState: Managed
+   storageClassDevices:
+     - devicePaths:
+         - /dev/vda
+         - /dev/vdb
+       storageClassName: assisted-service
+       volumeMode: Filesystem
+----
++
+You can use the following command to check the availability of the PVs:name: value
```

Comment:
The particular devicePaths used here are just an example, and the effective configuration may depend on the real hw used

Reply:
I will re-phrase to mention its an example